### PR TITLE
added screen shots of the current state of app

### PR DESCRIPTION
**Pull request by aparsh**
image

### CODE REVIEW

**Review by chanmol1999 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -61,3 +61,12 @@ It is advisable to clone only the development branch using the following command
 * Make all the PR's in development branch.
 * Contact the mentors at Gitter or Slack channel before contributing.
 
+#### Current state of app
```
Comment (atm1504) [inline README.md:64]: Increase the size of the images. Show around 3 to 4 images in a row.

**Review by atm1504 — CHANGES_REQUESTED** [review]

**Review by atm1504 — CHANGES_REQUESTED** [review]
Make different pr for this part. Use different branch other then development

**Comment by atm1504** [comment]
@aparsh don't make pr from development branch. Always make different branches for solving different issues. development should be synced with project development. You should not harm it any other way.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -12,6 +12,7 @@
         <activity
             android:name=".Dictionary.DictionaryActivity"
             android:label="Dictionary"
+            android:windowSoftInputMode="stateHidden"
```
Comment (atm1504) [inline app/src/main/AndroidManifest.xml]: Make different pr for solving this part of the issue.

**Review by atm1504 — CHANGES_REQUESTED** [review]
